python/cugraph/cugraph/comms/comms.py

```diff
@@ -26,10 +26,10 @@
 
 
 def __get_2D_div(ngpus):
-    pcols = int(math.sqrt(ngpus))
-    while ngpus % pcols != 0:
-        pcols = pcols - 1
-    return int(ngpus/pcols), pcols
+    prows = int(math.sqrt(ngpus))
+    while ngpus % prows != 0:
```

Comment:
this will be a very fast loop, but you should have just been able to subtract the modulo result from prows rather than looping.

Reply:
Can we do this? I am not sure I am fully understanding your intention, but here `prows` is varying.

Say `ngpus` = 14. Then, `int(sqrt(14))` will be 3. `14 % 3` will be 2. `prows (3) - 2` will be 1. But we want `prows` to be 2.

This code is based on the discussions from https://stackoverflow.com/questions/16266931/input-an-integer-find-the-two-closest-integers-which-when-multiplied-equal-th